cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc

```diff
@@ -160,8 +160,8 @@ RETURN node.title, score
 2+d|4 rows
 |===
 
-The fulltext schema indexes uses Lucene under the hood.
-This means that we can use Lucene's fulltext query language to express what we wish to search for.
+The full-text schema indexes uses Lucene under the hood.
```

Comment:
Similar to the previous comment - I suggest changing this to something similar to:

Full-text schema indexes are powered by the http://lucene.apache.org/[Apache Lucene] indexing and search library.

Reply:
Ditto! Thank you @mariascharin!